build/container/Dockerfile

```diff
@@ -59,6 +59,9 @@ ENV NRIA_IS_SECURE_FORWARD_ONLY true
 #################################
 FROM forwarder AS k8s-events-forwarder
 
+RUN apk add --no-cache \
+        curl=7.77.0-r0
```

Comment:
can we extract the version into a var so we don't need to update it twice each time?

Reply:
done ✅